src/physics/base/GeantImporter.cc

```diff
@@ -23,7 +23,10 @@ namespace celeritas
 /*!
  * Default constructor and destructor
  */
-GeantImporter::GeantImporter()  = default;
+GeantImporter::GeantImporter(std::string filename)
+{
+    this->root_input_.reset(TFile::Open(filename.c_str(), "read"));
+}
```

Comment:
Since `std::string` isn't being modified or moved, you should either pass `const std::string&` or just `const char*` since that's the only thing used by the function.